django/otto/settings.py

```diff
@@ -98,7 +98,10 @@
 OPENAI_EMBEDDING_COST_PER_TOKEN = 0.0004 / 1000
 
 DEFAULT_CHAT_MODEL = "gpt-4o-mini"
-USD_TO_CAD = 1.36
+DEFAULT_QA_MODEL = "gpt-4o-mini"
+DEFAULT_SUMMARIZE_MODEL = "gpt-4o-mini"
+DEFAULT_TRANSLATE_MODEL = "gpt-4o-mini"
+USD_TO_CAD = 1.38
```

Comment:
Is there a better way of doing this besides periodically re-hard-coding the exchange rate? Can we maybe link it up to some kind of live tracker?